### DISCUSSION TITLE
About sel key settting when training multiple systems

### DISCUSSION THREAD

**Discussion by shuaijiang-ustc** [deepmodeling/deepmd-kit#259] [Q&A] [answered]
Dear deepmd developers,

Thanks for sharing with us the wonderful DPMD kit. 
Recently I intended to train multiple systems with different elements compositions. Let's say A and B system for instance. According to the previous answers, the "system" key in json file was set.  The "type_map" key was set to include all the elemets in multiple systems. However, the "sel" key setting is not clear to me. 
"sel":		[[10, 6, 0, 4, 0], [7, 2, 1, 0, 0]],  is not working. 
Any suggestions are welcome. Thanks again.

Best,

Shuai

**Comment by njzjz** [comment, marked as answer]
Please use one list for them. For example, you can set 10 elements, the first 5 elements for the first system (`0, 1, 2, 3, 4`), and the last 5 elements for the second system (`5, 6, 7, 8, 9`). You can also combine the same elements.

**Comment by shuaijiang-ustc** [comment]
Thank you very much!
